docs/core/introduction.md

```diff
@@ -179,231 +76,172 @@ A .NET app is built from source code by using [MSBuild](/visualstudio/msbuild/ms
+It can also be built and run as two separate steps. The following example is for an app that is named *app*:
 
-For more information, see [NuGet documentation](/nuget/).
+```bash
+% dotnet build
+% ./bin/Debug/net6.0/app 
+Hello, World!
+```
 
-### .NET Interactive
+## Binary distributions
 
-.NET Interactive is a group of CLI tools and APIs that enable users to create interactive experiences across the web, markdown, and notebooks.
+You can [download .NET](https://dotnet.microsoft.com/download/dotnet/6.0) for development or production scenarios:
 
-For more information, see the following resources:
+* [.NET SDK](sdk.md) -- Set of tools, libraries, and runtimes for development, building, and testing apps.
+* [.NET Runtimes](https://dotnet.microsoft.com/en-us/download/dotnet/6.0/runtime) -- Set of runtimes and libraries, for running apps. Required for [framework-dependent apps](#deployment-models).
 
-* [.NET In-Browser Tutorial](https://dotnet.microsoft.com/learn/dotnet/in-browser-tutorial/1)
-* [Using .NET notebooks with Jupyter on your machine](https://github.com/dotnet/interactive/blob/main/docs/NotebookswithJupyter.md)
-* [.NET Interactive documentation](https://github.com/dotnet/interactive/blob/main/docs/README.md)
-
-## Execution models
-
-.NET apps run [managed code](../standard/managed-code.md) in a runtime environment known as the Common Language Runtime (CLR).
+## Free and open source
 
-### CLR
+.NET is free, open source, and is a [.NET Foundation](https://dotnetfoundation.org/) [project](https://dotnetfoundation.org/projects/netcore/). .NET is maintained by Microsoft and the community on GitHub in [several repositories](https://github.com/dotnet/core/blob/main/Documentation/core-repos.md).
 
-The .NET [CLR](../standard/clr.md) is a cross-platform runtime that includes support for Windows, macOS, and Linux. The CLR handles memory allocation and management. The CLR is also a virtual machine that not only executes apps but also generates and compiles code using a just-in-time (JIT) compiler.
+.NET source and and binaries are licensed with the [MIT license](https://github.com/dotnet/runtime/blob/main/LICENSE.TXT). Additional [licenses apply on Windows](https://github.com/dotnet/core/blob/main/license-information-windows.md) for binary distributions.
 
-For more information, see [Common Language Runtime (CLR) overview](../standard/clr.md).
+You can get the .NET SDK and runtimes from the following sources:
```

Comment:
source and binaries? little unclear what the difference is between these and the binaries links above.